src/main/java/com/google/vinet/data/TimeSlot.java

```diff
@@ -33,6 +33,7 @@
 
   protected TimeSlot(Instant start, Instant end, RegisteredUser registeredUser) {
     if (start == null || end == null) throw new NullPointerException();
+    System.out.println("Start: " + start.toString() + " end: " + end.toString());
```

Comment:
Intentional or for testing purposes?

Reply:
Sorry this PR wasn't actually ready for review yet. It is missing some commits due to merge conflicts.